forms-web-app/src/lib/session.js

```diff
@@ -0,0 +1,34 @@
+const connectMongodb = require('connect-mongodb-session');
+const session = require('express-session');
+const logger = require('./logger');
+const config = require('../config');
+
+module.exports = () => {
+  const { sessionSecret } = config.server;
+
+  if (!sessionSecret) {
+    throw new Error('Session secret must be set');
+  }
+
+  const MongoDBStore = connectMongodb(session);
```

Comment:
maybe worth wrapping in a try / catch ?

maybe not, idk, just a suggestion

Reply:
Personally, I want it to fail fast. If it can't connect, I want it to crash the system so it's (perversely) more robust - that way, K8S will remove the pod and spin up another